HubTurboUpdate.json

```diff
@@ -0,0 +1,12 @@
+{
+  "listOfLinks": [
```

Comment:
We can be a bit more specific about what links these are. Known versions we haven't downloaded?